x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -36,6 +37,12 @@ import { getMetrics, PerformanceMetrics } from './metrics';
 
 interface CreatePageOptions {
   browserTimezone?: string;
+  defaultViewport?: {
```

Comment:
Should this be a required property instead of optional? It looks like the only area it's not given is in tests.

Reply:
Yeah, let's do that! I'll also make both `height` and `width` required values!

Reply:
Great! If it doesn't impact anything (existing POST URLs?), it would make things more clear.